examples/step-16/doc/results.dox

```diff
@@ -77,7 +77,14 @@ implement because it needs all those additional data structures, and
 it becomes even more difficult if the program is to run in %parallel on
 machines coupled through MPI, for example. In that case, it would be
 simpler if one could use a black-box preconditioner that uses some
-sort of multigrid hierarchy for good performance but can figure out
+sort of multigrid hierarchy for good performance but can figure
 level matrices and similar things out by itself. Algebraic multigrid
```

Comment:
Not a very important point, but since most dictionaries list "figure out" as a verb I believe the 'out' should go at just the beginning and not at the end (i.e., use "but can figure out level matrices and similar things by itself").

Yes, I did have to look this one up.